specification/maps/data-plane/Route/preview/2024-07-01-preview/route.json

```
@@ -0,0 +1,2874 @@
+      "type": "object",
+      "properties": {
+        "longName": {
+          "description": "The long name of an address, such as Washington.",
+          "type": "string",
+          "readOnly": true
+        },
+        "distanceInMeters": {
+          "description": "The total distance traveled in meters within the administrative district.",
+          "type": "number",
+          "readOnly": true
+        },
+        "durationInSeconds": {
+          "description": "Estimated travel time in seconds that does not include delays on the route due to traffic conditions.",
+          "format": "int64",
+          "type": "integer",
+          "readOnly": true
+        },
+        "durationTrafficInSeconds": {
+          "description": "The time that it takes, in seconds, to travel a corresponding `TravelDistance` with current traffic conditions. This value is provided if `optimizeRoute` includes traffic considerations.",
+          "format": "int64",
+          "type": "integer",
+          "readOnly": true
+        }
+      }
+    },
+    "Leg": {
+      "x-ms-client-name": "RouteLeg",
+      "description": "Information about a section of a route between two waypoints. More information about the fields contained in a route Leg:",
+      "type": "object",
+      "properties": {
+        "description": {
+          "description": "A short description of the route.",
+          "type": "string",
+          "readOnly": true
+        },
+        "routePathRange": {
+          "$ref": "#/definitions/RoutePathRange"
+        },
+        "distanceInMeters": {
+          "description": "The physical distance in meters covered by a route Leg.",
+          "type": "number",
+          "readOnly": true
+        },
+        "durationInSeconds": {
+          "description": "Estimated travel time in seconds that does not include delays on the route due to traffic conditions.",
+          "format": "int64",
+          "type": "integer",
+          "readOnly": true
+        },
+        "durationTrafficInSeconds": {
+          "description": "The time that it takes, in seconds, to travel a corresponding `TravelDistance` with current traffic conditions. This value is provided if `optimizeRoute` includes traffic considerations.",
+          "format": "int64",
+          "type": "integer",
+          "readOnly": true
+        },
+        "departureTime": {
+          "description": "The estimated departure time for the leg, which takes into account the traffic conditions, is formatted as a dateTime value defined by [RFC 3339, section 5.6](https://www.rfc-editor.org/rfc/rfc3339#section-5.6). It will reference the timezone offset by either `departAt` or `arrivalAt`. If not, then the UTC time will be used.",
+          "format": "date-time",
```

Here too. Why were these removed?
```suggestion
          "type": "string",
          "format": "date-time",
```